### PR TITLE
Feature/guild pages

### PR DESCRIPTION
**Pull request by mquellhorst**
The following data is needed to fill in the required information

guild.twitter_url
guild.github_url
guild.reddit_url ??
guild.mastodon_url ??
guild.Players
guild.quests

Guild should be able to define their NFT gallery address

We need to define what we will place in the Stats block

### CODE REVIEW

File: packages/web/components/Guild/GuildTile.tsx [hunk 1]
```diff
@@ -46,14 +51,6 @@ export const GuildTile: React.FC<Props> = ({ guild }) => (
...
     </MetaTileHeader>
     <MetaTileBody>
-      <VStack spacing={2} align="stretch">
-        <Text fontFamily="mono" fontSize="sm" color="blueLight">
-          LINKS
-        </Text>
-        <HStack mt="2">
-          <GuildLinks guild={guild} />
-        </HStack>
-      </VStack>
```
Comment (alalonde) [inline packages/web/components/Guild/GuildTile.tsx:56]: What is the rationale for removing the links ?

File: packages/web/components/Guild/GuildLinks.tsx [hunk 2]
```diff
@@ -1,44 +1,75 @@
...
+                size="lg"
+                bgColor="discord"
+                icon={<FaDiscord />}
+              />
+            </a>
+          </WrapItem>
+        ) : null}
+        {guild.website_url ? (
+          <WrapItem>
+            <a href={guild.website_url} target="_blank" rel="noreferrer">
```
Comment (alalonde) [inline packages/web/components/Guild/GuildLinks.tsx:48]: wrong link. Let's not show this if a github URL is not available

File: packages/web/components/Guild/GuildLinks.tsx [hunk 3]
```diff
@@ -1,44 +1,75 @@
...
+              <IconButton
+                variant="outline"
+                aria-label="Discord Server"
+                size="lg"
+                colorScheme="blackAlpha"
+                icon={<FaGlobe />}
+              />
+            </a>
```
Comment (alalonde) [inline packages/web/components/Guild/GuildLinks.tsx:26]: Personally, I prefer the previous styling

**Review by alalonde — CHANGES_REQUESTED** [review]

File: packages/web/components/Guild/GuildTile.tsx (hunk 1 above)
Comment (mquellhorst) [reply packages/web/components/Guild/GuildTile.tsx:56]: Cause the links are in a different place on the guild details page

File: packages/web/components/Guild/GuildLinks.tsx (hunk 2 above)
Comment (mquellhorst) [reply packages/web/components/Guild/GuildLinks.tsx:48]: yeah I know, I put this in temporarily cause the fields aren't available yet

File: packages/web/components/Guild/GuildLinks.tsx (hunk 3 above)
Comment (mquellhorst) [reply packages/web/components/Guild/GuildLinks.tsx:26]: This is a lot closer to the styling from the designs

File: packages/web/components/Guild/GuildTile.tsx (hunk 1 above)
Comment (alalonde) [reply packages/web/components/Guild/GuildTile.tsx:56]: Seems useful to have them in the tiles at-a-glance as well ? There isn't a whole lot of info at the moment on a particular tile

File: packages/web/components/Guild/GuildLinks.tsx (hunk 3 above)
Comment (alalonde) [reply packages/web/components/Guild/GuildLinks.tsx:26]: Is the design in that figma document?

Comment (mquellhorst) [reply packages/web/components/Guild/GuildLinks.tsx:26]: yeah

File: packages/web/components/Guild/GuildTile.tsx (hunk 1 above)
Comment (mquellhorst) [reply packages/web/components/Guild/GuildTile.tsx:56]: Yeah well we could keep them in the GuildHero than we need to update the design.
A guild could have guite a lot of links tho which makes the hero a bit too messy

File: packages/web/components/Guild/GuildLinks.tsx (hunk 2 above)
Comment (alalonde) [reply packages/web/components/Guild/GuildLinks.tsx:48]: OK, we don't want to show the github link is website_url is set though.

**Comment by alalonde** [comment]
Sorry for the delay on this. Can you rebase and make those changes, then we'll merge it in

**Review by alalonde — APPROVED** [review]
Rebased and added two missing fields for guilds.

**Pull request merged**